docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -32,53 +32,83 @@ This function is useful for removing global state or closing a server that was u
 TIP: {playwright-url}[Playwright] is browser testing library developed by Microsoft.
 It is reliable and fast and features a modern API that auto waits for page elements to be ready.
 
+// [discrete]
+// [[synthetics-install]]
+// == Install Elastic Synthetics
+
+// Create a new https://docs.npmjs.com/cli/v7/commands/npm-init[NPM/Node.js project].
+// Then, install the synthetics runner as a dev-dependency using `npm install -D @elastic/synthetics`.
+
 [discrete]
 [[synthetics-create-journey]]
-=== Create a `journey`
+== Write a synthetic test
 
-The `journey` function takes a single parameter, `name`,
-and provides access to `page`, `params`, `browser`, and `context`:
-
-* `browser` -- A {playwright-api-docs}[browser] object created by Playwright.
-* `page` -- A https://playwright.dev/docs/api/class-page[page] object from Playwright that lets you control the browser's current page.
-* `context` -- A https://playwright.dev/docs/api/class-browsercontext[browser context] that doesn't share cookies or cache with other browser contexts.
-* `params` -- User-defined variables that allow you to invoke the Synthetics suite with custom parameters.
-For example, if you want to use a different homepage depending on the `env`
-(localhost for `dev`, and a URL for `prod`). See <<synthetics-params-secrets>>
-for more information.
+A journey tests one discrete unit of functionality.
+For example, logging into a website, adding something to a cart, or joining a mailing list.
 
-Putting it all together, a basic, two step journey might look something like this:
+Start by creating a new file called using the `.journey.ts` or `.journey.js` file extension,
+importing the synthetics library, and adding a new journey.
 
 [source,js]
 ----
-journey("Journey name", ({ page }) => {
-    step("Step 1 name", () => {
-      // Do something here
-    })
-    step("Step 2 name", () => {
-      // Do something else here
-    })
+journey('Journey name', ({ page, browser, context, params }) => {
+  // Add steps here
 });
 ----
 
+The `journey` function takes a single parameter, `name` ('Journey name' in the example above).
+Then you pass a callback to the global `journey` function to encapsulate the whole test.
```

Comment:
I think that saying that the `journey` function takes a single parameter is inaccurate. It actually takes two: an `options` object (which can include `name` and `id` for now), and a callback. However, for now it may be worth not mentioning the `options` object and just mentioning `name` instead.

Also, following it with  "Then you pass [...]" may make this sound like a sequential action. Maybe using the following could be clearer:

> The `journey` function takes two parameters: a `name`, and a `callback`. The `name` helps you identify an individual journey. The `callback` argument is a function which encapsulates what the journey does.

Later we could add more explanation about how we link results to journey's IDs and how users could use IDs vs. (or plus) names. What do you think @andrewvc? Should we omit info about `id` for now?